src/App.js

```diff
@@ -1,21 +1,36 @@
 import React, {Component} from 'react';
+import Cookies from 'js-cookie'
 
 let audience = "stubOidcClient";
 let redirectTo = "http://localhost:5000/callback";
 let oidcProviderBaseUrl = 'http://localhost:9000';
+let dittNavHendelserUrl = 'http://localhost:9002?hendelser';
 let redirectToInitTokenFlow = oidcProviderBaseUrl + "/auth?client_id=" + audience + "&redirect_uri=" + redirectTo + "&response_type=code&scope=openid+profile+acr+email&nonce=123";
 let clientSecret = "secretsarehardtokeep";
 let authenticationHeader = new Buffer(audience + ":" + clientSecret).toString('base64');
 
 class App extends Component {
     state = {
-        idToken: ""
+        idToken: "",
     };
 
     redirectToAuthenticationPage(sikkerhetsnivaa) {
         window.location.assign(`${redirectToInitTokenFlow}&acr_values=${sikkerhetsnivaa}`);
     }
 
+    redirectToDittNav() {
+        window.location.assign(`${dittNavHendelserUrl}`);
+    }
+
+    setCookie() {
+        let hasToken = this.state.idToken !== "";
+        if (hasToken) {
```

Comment:
Dette kan forkortes til if(this.state.idToken) :)

Reply:
👍